src/_pytest/runner.py

```diff
@@ -278,10 +278,7 @@ def __init__(self):
         self._finalizers = {}
 
     def addfinalizer(self, finalizer, colitem):
-        """ attach a finalizer to the given colitem.
-        if colitem is None, this will add a finalizer that
-        is called at the end of teardown_all().
-        """
+        """ attach a finalizer to the given colitem. """
```

Comment:
this migght be a breaking api change, lets crosscheck

Reply:
This docstring is outdated -- see the assert below `assert colitem and not isinstance(colitem, tuple)`. The change updates the docstring to remove the outdated info.